docs/streamlit_components.md

```diff
@@ -27,6 +29,22 @@ Check out this [Streamlit Components Tutorial video](https://www.youtube.com/wat
   </iframe></div>
 ```
 
+[Part 2 - Make a Slider Widget](https://www.youtube.com/watch?v=QjccJl_7Jco)
+
+```eval_rst
+.. raw:: html
+
+  <div class='embed-container'><iframe
+    width="560"
+    height="315"
+    src="https://www.youtube.com/embed/QjccJl_7Jco"
+    style="margin: 0 0 2rem 0;"
+    frameborder="0"
+    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+    allowfullscreen>
+  </iframe></div>
+```
+
 ```eval_rst
 .. note::
    The remainder of the documentation in this section is for users
```

Comment:
This section no longer makes sense, since there's no documentation below. My edit:

Reply:
```eval_rst
.. note::
   If you are only interested in **using Streamlit Components**, then you can skip this section and head over to the `Streamlit Components Gallery <https://streamlit.io/components>`_ to find and install components created by the community!
```

I'd also bump this to the top of the page.

Reply:
Agreed